### ISSUE TITLE
XBMC on A10

### ISSUE THREAD

**Issue by s-kostyuk** [cubieplayer/Cubian#223]
Good day. Have anyone tried to compile and run XBMC on Cubieboard 1? If so, how to improve mediacenter performance? I used this instruction for build and install XBMC itself: http://linux-sunxi.org/XBMC and this for libve: http://linux-sunxi.org/CedarX/libve and got "slideshow" for almost two videos:
1) 24 frames/second, H264 - MPEG4 AVC, 960x400 pixels, mkv
2) 24 frames/second, MPEG4 (XVID), 720x384 pixels, mkv

Kernel: Linux Cubian 3.4.67+ #18 PREEMPT Wed Nov 6 08:26:36 CST 2013 armv7l GNU/Linux

**Comment by cubieplayer** [comment]
Yes, actually i had made an image but not released yet, everthing works perfect.

**Comment by luoyi** [comment]
hello, cubieplayer, does your xbmc for A10 is installable from the cubian repo ?

**Comment by toketin** [comment]
Cubieplayer, please release it soon! I've tried XBMC, compiled without X support, under Archlinux Arm, but it's laggy with 720p videos and there's no sound. If you said that with your image, everything works perfect, i'm curious to see it on my CB1. Thanks!

**Comment by s-kostyuk** [comment]
Any news?
